binding.gyp

```diff
@@ -43,5 +43,48 @@
         ]
       }]
     ]
+  },
+  {
+    "target_name": "appsec",
+    "sources": [
+      "packages/dd-trace/src/appsec/native/src/main.cpp"
+    ],
+    "include_dirs": [
+      "<!@(node -p \"require('node-addon-api').include\")",
+      "<!(node -p \"require('./packages/dd-trace/src/appsec/native/lib/lib.js').include\")"
+    ],
+    "libraries": [
+      "<!(node -p \"require('./packages/dd-trace/src/appsec/native/lib/lib.js').lib\")"
+    ],
+    "defines": [ "NAPI_DISABLE_CPP_EXCEPTIONS" ],
+    "xcode_settings": {
+      "MACOSX_DEPLOYMENT_TARGET": "10.10",
+      "OTHER_CFLAGS": [
+        "-std=c++14",
+        "-stdlib=libc++",
+        "-Wall",
+        "-Werror"
+      ]
+    },
+    "conditions": [
+      ["OS == 'linux'", {
+        "cflags": [
+          "-std=c++11",
+          "-Wall",
+          "-Werror"
+        ],
+        "cflags_cc": [
+          "-Wno-cast-function-type"
+        ]
+      }],
+      ["OS == 'win'", {
+        "libraries": [
+          "Ws2_32.lib"
```

Comment:
I am not sure if adding a `libraries` entry here in the windows condition replaces the top level one, or gets merged with it (L56)

Reply:
not sure either, can you debug by checking the vsproj locally ?

Reply:
Yes I tried it and it all works, so my guess is we're good.